XAlloc.h

```diff
@@ -35,4 +35,10 @@ void* xRealloc(void* ptr, size_t size);
 
 char* xStrdup_(const char* str) ATTR_NONNULL;
 
+static inline void* xFree(void* ptr) {
+   if (ptr)
+      free(ptr);
```

Comment:
`free(NULL);` is no-op as per the C standard, no check needed.
Cf. https://stackoverflow.com/a/1938758

Runtimes that crash here are broken in the first place and should be fixed instead.

Reply:
dropped check